web/src/components/molecules/Common/Header/profile.tsx

```diff
@@ -92,6 +92,9 @@ const HeaderProfile: React.FC<Props & Partial<LoginProps>> = ({
             <MenuListItem>
               <MenuListItemLabel icon="logout" onClick={onSignOut} text={t("Log out")} />
             </MenuListItem>
+            <MenuListItem>
+              <MenuListItemLabel text={t(`v${__APP_VERSION__}`)} />
```

Comment:
I don't think this needs to be wrapped with the `t` function. At least for Japanese and english (and probably many languages) software stuff like v for version will not change

Reply:
Oh yeah you're right!